FAQ.md

```diff
@@ -45,6 +48,14 @@ A: The wallet tab can be found after clicking the hamburger menu in the top left
 Q: How can I back up my ItchySats wallet?\
 A: On Umbrel the wallet is derived from the Umbrel app-seed, so you don't need to backup anything in addition. Binary and docker containers users should backup the `taker_seed` file that is used to derive the wallet.
 
+## Seed
+
+Q: What's `taker_seed`?
+A: A file that is used to derive both the wallet keys and the credentials for the ItchySats UI. It is generated on the first launch, It is essential to back it up and store it securely, especially when using ItchySats on mainnet.
+
+Q: Can I restore my ItchySats balance and trade history if I copy `taker_seed` again?
+A: Certainly. New `taker_seed` file is only generated if there is none present; if you overwrite a new seed with an old one, you should see you balance again after you start ItchySats again. It is recommended to go through the backup/restore procedure at least once to be safe that nothing is lost.
```

Comment:
By only backing up the `taker_seed` a user want be able to recover their trade history though.